TechnicoServer/TechnicoServer/Models/RepairCategory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
I wouldn't use different model just for the Repair Type. Consider of using only Repair.cs.